src/NHibernate/Linq/DBOnlyAttribute.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+
+namespace NHibernate.Linq
+{
+	/// <summary>
+	/// Can flag a method as not being callable by the runtime, when used in Linq queries.
+	/// If the method is supported by the linq-to-nhibernate provider, and <see cref="DBOnly"/>
+	/// is <see langword="true" />, it will always be converted to the corresponding SQL statement.
+	/// Otherwise the linq-to-nhibernate provider evaluates method calls when they do not depend on
+	/// the queried data. <see cref="DBOnly"/> default value is <see langword="true" />.
+	/// </summary>
+	public class DBOnlyAttribute : Attribute
```

Comment:
Stating the default value in class comment instead of on property, since this property changes its default value when inherited.